src/Uplink/Messages/Expired_Key.php

```diff
@@ -1,20 +1,81 @@
 <?php
 
 namespace StellarWP\Uplink\Messages;
+use StellarWP\Uplink\Config;
 
 class Expired_Key extends Message_Abstract {
 	/**
 	 * @inheritDoc
 	 */
 	public function get(): string {
-        $message  = '<div class="notice notice-warning"><p>';
-        $message  .= __( 'Your license is expired', '%TEXTDOMAIN%' );
-		$message .= '<a href="https://evnt.is/195y" target="_blank" class="button button-primary">' .
-			__( 'Renew Your License Now', '%TEXTDOMAIN%' ) .
-			'<span class="screen-reader-text">' .
-			__( ' (opens in a new window)', '%TEXTDOMAIN%' ) .
-			'</span></a>';
-        $message .= '</p>    </div>';
+		// TEC only default link for backwards compatibility.
+		$default_link = in_array(
+			Config::get_hook_prefix(),
+			[
+				'the-events-calendar',
+				'events-calendar-pro',
+				'event-tickets',
+				'event-tickets-plus',
+				'tribe-filterbar',
+				'events-virtual',
+				'events-community',
+				'events-community-tickets',
+				'event-aggregator',
+				'events-elasticsearch',
+				'image-widget-plus',
+				'advanced-post-manager',
+				'tribe-eventbrite',
+				'event-automator',
+				'tec-seating',
+			],
+			true ) ? 'https://evnt.is/195y' : '';
+
+		$message_link        = apply_filters( 'stellarwp/uplink/' . Config::get_hook_prefix() . '/messages/expired_key_link', $default_link );
+		$renew_label         = __( 'Renew Your License Now', '%TEXTDOMAIN%' );
+		$opens_in_new_window = __( '(opens in a new window)', '%TEXTDOMAIN%' );
+		$notice_text         = __( 'Your license is expired', '%TEXTDOMAIN%' );
+		
+		if ( ! empty( $message_link ) ) {
+			$message_content = sprintf(
+				'<p>%s <a href="%s" target="_blank" class="button button-primary">%s <span class="screen-reader-text">%s</span></a></p>',
```

Comment:
Should we add `// translators: %s`? Other than that, should we add order placeholders (`%1$s`)?